### ISSUE TITLE
Upgrade to Django 2.1

### ISSUE THREAD

**Issue by bfirsh** [embassynetwork/modernomad#441]
Just gonna file this here as a low-priority thing that needs doing. :)

`python -Wa manage.py test` throws up lots of warnings that hint at what needs doing. Stuff that can be fixed now, in fact.

Blocked by #339

**Comment by jonathan-s** [comment]
Assigning myself to this, I generally think we should launch the rebooted version before this happens.

**Comment by bfirsh** [comment]
Indeed. In an ideal world we would have deployed all the new Django 1.11 and Python 3 stuff to the old server first to minimise the variables in the move, but oh well. ;)
